Instructions for creating custom images are incorrect

**Describe the bug**
The documentation describing how to setup your own dockerfile based on the web ready image are incorrect. During the entrypoint execution we expect the JDBC driver to be in the /opt/pega/lib directory so we can use it to check the specific Pega Version. This was a new requirement added as part of support for FIPS 140-3 mode.

If a user puts the JDBC driver directly in the tomcat/lib directory, the version check fails due to a missing driver:
```
executing command: java -cp '/opt/pega/utility/*:/opt/pega/lib/*' -Xms2048m -Xmx4096m -Djavax.net.ssl.trustStore=/truststore/cacerts -Duser.timezone=America/New_York  --add-modules java.se --add-exports java.base/jdk.internal.ref=ALL-UNNAMED --add-opens java.base/java.lang=ALL-UNNAMED --add-opens java.base/java.nio=ALL-UNNAMED --add-opens java.base/sun.nio.ch=ALL-UNNAMED --add-opens java.management/sun.management=ALL-UNNAMED --add-opens jdk.management/com.ibm.lang.management.internal=ALL-UNNAMED --add-opens jdk.management/com.sun.management.internal=ALL-UNNAMED com.pega.cmc.VersionChecker /opt/pega/utility/platform_version.txt
WARNING: package com.ibm.lang.management.internal not in jdk.management
com.pega.cmc.VersionCheckerException: Encountered exception while trying to load 'org.postgresql.Driver' driver class.
	at com.pega.cmc.VersionChecker.loadDriver(VersionChecker.java:63)
	at com.pega.cmc.VersionChecker.checkVersion(VersionChecker.java:127)
	at com.pega.cmc.VersionChecker.main(VersionChecker.java:156)
Caused by: java.lang.ClassNotFoundException: org.postgresql.Driver
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:525)
	at java.base/java.lang.Class.forName0(Native Method)
	at java.base/java.lang.Class.forName(Class.java:469)
	at com.pega.cmc.VersionChecker.loadDriver(VersionChecker.java:60)
	... 2 more
Pega version check failed and container will exit -- verify your database connection details.
To bypass this check, set global.pegaVersion in the Pega Helm Chart values.yaml file.  The value
should be set to the Pega version in the [major].[minor].[patch] format.
```

**To Reproduce**
Follow the instructions in the readme and try to run the container.

**Expected behavior**
Documentation should instruct users to copy the jar into /opt/pega/lib so we pick it up on the classpath when running the Version check. The entrypoint will copy the contents of that directory to tomcat/lib later in the execution.

--- a/README.md
+++ b/README.md
@@ -103,7 +103,7 @@ COPY --chown=pegauser:root --from=builder /prweb ${CATALINA_HOME}/webapps/prweb
 RUN chmod -R g+rw   ${CATALINA_HOME}/webapps/prweb
 
 # Make a jdbc driver available to tomcat applications
-COPY --chown=pegauser:root /path/to/jdbcdriver.jar ${CATALINA_HOME}/lib/
+COPY --chown=pegauser:root /path/to/jdbcdriver.jar /opt/pega/lib/
 
 RUN chmod g+rw ${CATALINA_HOME}/webapps/prweb/WEB-INF/classes/prconfig.xml
 RUN chmod g+rw ${CATALINA_HOME}/webapps/prweb/WEB-INF/classes/prlog4j2.xml

